### PR TITLE
Add variable order optional parameter

### PR DESCRIPTION
**Pull request by Kaasiak**
Solves i#22

### CODE REVIEW

File: tests/test_event.py [hunk 1]
```diff
@@ -58,6 +62,9 @@ def test_check_sampling_zero_get_and_set(self) -> None:
         szp = [('Medium',), ('Medium', 'High')]
         self.et.sampling_zeros = szp
         assert self.et.sampling_zeros == szp
+    
+    def test_order_of_columns(self) -> None:
+        assert self.reordered_et.variables == self.df.coumns[::-1]
```
Comment (g-walley) [inline tests/test_event.py:67]: This line appears to be ambiguous. self.df.columns (misspelled) is not a list, and doesn't appear to have the "==" operator defined for comparing the columns datatype to a list. I recommend updating this line to the following:
`assert self.reordered_et.variables == list(self.df.columns[::-1])`

**Review by g-walley — CHANGES_REQUESTED** [review]

File: tests/test_event.py (hunk 1 above)
Comment (Kaasiak) [reply tests/test_event.py]: Resolved. For the future, how can I run the unit tests to check everything runs smoothly before pushing to remote?

Comment (g-walley) [reply tests/test_event.py]: We use pytest with a few plugins. If you could install the following packages:
pytest
pytest-mock
pytest-xdist
pytest-cov

Then from the terminal, go to the project folder and run:
pytest tests

**Review by g-walley — APPROVED** [review]
Looks good! I'll merge this today :)

**Pull request merged**